File: src/routes/trx.js
const routering = require('express').Router();
const currentHostRunsExecutions = require('../controllers/trx')

routering.get('/trx',(req,res)=>{
    currentHostRunsExecutions.listarTrx(req,res);
})

routering.get('/trxLasteSixMoth',(req,res)=>{
    currentHostRunsExecutions.listarSixMonht(req,res);
})

module.exports = routering

File: src/routes/trx.py
from flask import Blueprint, request, jsonify
from controllers.trx import listarTrx, listarSixMonth

routerTrx = Blueprint('routering', __name__)

@routerTrx.route('/trx', methods=['GET'])
def trx_route():
    return listarTrx(request, jsonify)

@routerTrx.route('/trxLasteSixMoth', methods=['GET'])
def six_month_trx_route():
    return listarSixMonth(request, jsonify)
